src/methods.js

```diff
@@ -11,153 +11,619 @@ import { map, set } from 'lodash';
+  },
+  preciousBlock: {
+    category: 'blockchain',
+    version: '>=0.14.0'
+  },
+  prioritiseTransaction: {
+    category: 'mining',
+    version: '>=0.10.0'
+  },
+  pruneBlockchain: {
+    category: 'blockchain',
+    version: '>=0.14.0'
+  },
+  removePrunedFunds: {
+    category: 'wallet',
+    features: {
+      multiwallet: '>=0.15.0'
+    },
+    version: '>=0.13.0'
+  },
+  sendFrom: {
+    category: 'wallet',
+    features: {
+      multiwallet: '>=0.15.0'
+    },
+    version: '>=0.3.18'
+  },
+  sendMany: {
+    category: 'wallet',
+    features: {
+      multiwallet: '>=0.15.0'
+    },
+    version: '>=0.3.21'
+  },
+  sendRawTransaction: {
+    category: 'rawtransactions',
+    version: '>=0.7.0'
+  },
+  sendToAddress: {
+    category: 'wallet',
+    features: {
+      multiwallet: '>=0.15.0'
+    },
+    version: '>=0.1.0'
+  },
+  setAccount: {
+    category: 'wallet',
+    features: {
+      multiwallet: '>=0.15.0'
+    },
+    version: '>=0.1.0'
+  },
+  setBan: {
+    category: 'network',
+    version: '>=0.12.0'
+  },
+  setGenerate: {
+    category: 'generating',
+    version: '<0.13.0'
+  },
+  setNetworkActive: {
+    category: 'network',
+    version: '>=0.14.0'
+  },
+  setTxFee: {
+    category: 'wallet',
+    features: {
+      multiwallet: '>=0.15.0'
+    },
+    version: '>=0.3.22'
+  },
+  signMessage: {
+    category: 'wallet',
+    features: {
+      multiwallet: '>=0.15.0'
+    },
+    version: '>=0.5.0'
+  },
   signMessageWithPrivKey: {
+    category: 'util',
     obfuscate: {
       request: params => set([...params], '[0]', '******')
     },
     version: '>=0.13.0'
   },
   signRawTransaction: {
+    category: 'rawtransactions',
     obfuscate: {
       request: params => set([...params], '[2]', map(params[2], () => '******'))
     },
     version: '>=0.7.0'
   },
-  stop: { version: '>=0.1.0' },
-  submitBlock: { version: '>=0.7.0' },
-  upTime: { version: '>=0.15.0' },
-  validateAddress: { version: '>=0.3.14' },
-  verifyChain: { version: '>=0.9.0' },
-  verifyMessage: { version: '>=0.5.0' },
-  verifyTxOutProof: { version: '>0.11.0' },
-  walletLock: { version: '>=0.1.0' },
+  stop: {
+    category: 'control',
+    version: '>=0.1.0'
+  },
+  submitBlock: {
+    category: 'mining',
+    version: '>=0.7.0'
+  },
+  upTime: {
+    category: 'control',
+    version: '>=0.15.0'
+  },
+  validateAddress: {
+    category: 'util',
+    version: '>=0.3.14'
+  },
+  verifyChain: {
+    category: 'blockchain',
+    version: '>=0.9.0'
+  },
+  verifyMessage: {
+    category: 'util',
+    version: '>=0.5.0'
+  },
+  verifyTxOutProof: {
+    category: 'blockchain',
+    version: '>0.11.0'
+  },
+  walletLock: {
+    category: 'wallet',
+    version: '>=0.1.0'
+  },
   walletPassphrase: {
+    category: 'wallet',
     obfuscate: {
       request: params => set([...params], '[0]', '******')
     },
     version: '>=0.1.0'
   },
-  walletPassphraseChange: { version: '>=0.1.0' }
+  walletPassphraseChange: {
```

Comment:
Please obfuscate the request passwords (`params[0]` and `params[1]`).

Reply:
Can I just open new merge request will all stuff fixes? Seems like original commit author is not there to do this.